modules/ROOT/pages/import-api-specification-maven.adoc

```diff
@@ -0,0 +1,63 @@
+= Importing an API Specification from Maven
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API specification in a Maven repository into a new Mule Project, or an existing one.
+
+This feature is possible through API Sync. If you are using an older version of Mule Runtime (4.1.3 and below), or if you don't want to use this new design paradigm and would still prefer to import an API specification that's not in Exchange, follow the xref:studio::import-api-specification-design-center.adoc[Importing an API Specification from a Local File] steps.
+
+== Importing an API From Maven Into a New Project
+
+. In the top task bar, select *File*, then *New*, and then *Mule Project*.
+. Type a name for your project.
+. Select a Mule runtime engine version 4.1.4 or later.
+. In the *API Specification* section, select the *From Exchange or Maven* tab, click the plus icon, and select *From Maven*. +
+You can either specify the Maven coordinates of your API specification, or you can install a local file into your Maven repository.
+
+// tag::maven-tasks[]
+=== Adding Your API From Maven Coordinates
+
+If your API specification is in your corporate's Maven repository, you can add it as a dependency.
+
+. In the *Import From Maven* screen, configure your API specification artifact definition:
+* Group ID.
+* Artifact ID.
+* Version.
+. In the *Advanced Settings* section, make sure that the *Classifier* of your dependency is the same as your API specification type (RAML, for example).
+. Click *OK*.
+
+Studio scaffolds your API specification. If you choose not to create flows out of your API specification, uncheck the *Scaffold Flows From These API Specifications* option.
+
+You can manage the API specifications in your project that are linked to Exchange from the API Project management view by clicking the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png[2%,2%]) in the top task bar.
```

Comment:
GLOBAL EDIT TO TEXT THROUGHOUT:

This is a separate task, right? Sort of a Next Steps item? Suggest this:

After you import them, you can manage the API specifications in your project that are linked to Exchange from the API project management view by clicking the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png[2%,2%]) in the top taskbar.